src/components/common/Pill.tsx

```diff
@@ -5,7 +5,7 @@ const Pill = ({ name, slug }: { name: string; slug: string }) => {
     <Link
       key={name}
       href={slug || ""}
-      className="max-content py-1 px-4 rounded-[5px] bg-gray-custom-700 hover:bg-gray-custom-600 hover:text-gray-custom-100 max-md:px-3 max-md:py-[2px] text-xs md:text-sm 2xl:text-base max-md:border max-md:border-gray-custom-300 max-md:leading-[100%]"
+      className="max-content first-letter:capitalize py-1 px-4 rounded-[5px] bg-gray-custom-700 hover:bg-gray-custom-600 hover:text-gray-custom-100 max-md:px-3 max-md:py-[2px] text-xs md:text-sm 2xl:text-base max-md:border max-md:border-gray-custom-300 max-md:leading-[100%]"
```

Comment:
we can remove the capitalize since we are using the topic name itself and not the slug